app/views/advertisements/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>All Advertisements</h1>
+<% @advertisements.each do |advertisement| %>
+  <div class="media">
+    <div class="media-body">
+      <h4 class="media-heading">
+        <%= link_to "$#{advertisement.price}: #{advertisement.title}",advertisement %>
```

Comment:
[number_to_currency](http://devdocs.io/rails/activesupport/numberhelper#method-i-number_to_currency) could be useful here